### PR TITLE
fix: add route to fetch build artifact

### PR DESCRIPTION
**Pull request by jithin1987**
## Objective

Add a route which issues a redirect to store with a temporary JWT to fetch artifacts.

@petey @stjohnjohnson

### CODE REVIEW

File: test/plugins/builds.test.js [hunk 1]
```diff
@@ -1593,4 +1601,23 @@ describe('build plugin test', () => {
             });
         });
     });
+
+    describe('GET /builds/{id}/artifacts/{artifcat}', () => {
```
Comment (tkyi) [inline test/plugins/builds.test.js]: artifact is mispelled

File: test/plugins/builds.test.js [hunk 2]
```diff
@@ -1593,4 +1601,23 @@ describe('build plugin test', () => {
...
+        const id = 12345;
+        const artifact = 'manifest';
+
+        it('rediects to store for a artifact request', () => {
```
Comment (tkyi) [inline test/plugins/builds.test.js]: redirects*

File: test/plugins/builds.test.js (hunk 1 above)
Comment (jithin1987) [reply test/plugins/builds.test.js]: fixed

File: test/plugins/builds.test.js (hunk 2 above)
Comment (jithin1987) [reply test/plugins/builds.test.js]: fixed

File: plugins/builds/artifacts/get.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+            'hapi-swagger': {
+                security: [{ session: [] }]
+            }
+        },
+        handler: (request, reply) => {
+            const artifact = request.params.name;
+            const buildId = request.params.id;
+
+            const token = jwt.sign({
+                buildId, artifact, scope: ['user']
```
Comment (minz1027) [inline plugins/builds/artifacts/get.js]: Why we sign with `buildId` and `artifact`? are we going to anything with this?

Comment (jithin1987) [reply plugins/builds/artifacts/get.js]: We are not doing anything for now/

**Pull request merged**